src/rank_llm/rerank/prompt_templates/rank_gpt_template.yaml

```diff
@@ -0,0 +1,11 @@
+method: "listwise_conv"
+system_message: "You are RankGPT, an intelligent assistant that can rank passages based on their relevancy to the query."
+prefix: |-
```

Comment:
since this is conversational change the keys to `prefix_user` and `body_user` and `suffix_user`